float32[] ee_vel
---
bool done